DigitalLearningSolutions.Data.Tests/Services/RecommendedLearningServiceTests.cs

```diff
@@ -235,32 +227,19 @@ public async Task GetRecommendedLearningForSelfAssessment_calls_learning_hub_api
         }
 
         [Test]
-        [TestCase(1, true, 100)]
-        [TestCase(1, false, 30)]
-        [TestCase(0, true, 0)]
+        [TestCase(true, 175)]
+        [TestCase(false, 105)]
         public async Task
-            GetRecommendedLearningForSelfAssessment_returns_correct_recommendation_score_for_resource_with_essential_question_parameters_only(
-                int numberOfQuestionParameters,
+            GetRecommendedLearningForSelfAssessment_returns_correct_recommendation_score_for_resource_with_essential_question_parameters(

```

Comment:
The test case where there is no CompetencyResourceAssessmentQuestionParameters record seems to have vanished (probably tied to the other comment about the `.Single()` vs `.SingleOrDefault()`)

Reply:
See also here https://github.com/TechnologyEnhancedLearning/DLSV2/pull/855#discussion_r783277920, the no record case doesn't really need other tests